src/energyplus/ForwardTranslator/ForwardTranslateWaterHeaterHeatPumpWrappedCondenser.cpp

```diff
@@ -130,11 +130,11 @@ namespace energyplus {
         if (auto thermalZone = modelObject.thermalZone()) {
           auto inletNode = modelObject.inletNode();
           OS_ASSERT(inletNode);
-          airInletNodeName = inletNode->name().get();
+          airInletNodeName = modelObject.airInletNodeName().value_or(inletNode->name().get());
```

Comment:
Should the optional air node names only be translated when inletAirConfiguration is "Schedule"? Or should we have rules for when air node names can be set based on current choice of inletAirConfiguration?